### DISCUSSION TITLE
unable to push to a forked repository

### DISCUSSION THREAD

**Discussion by sailod** [community/community#127931] [Repositories] [answered]
### Select Topic Area

Question

### Body

I get error when trying to push to a forked repository,
the error message indicate on some ban but I opened a ticket and its been over a month and nobody replied,
is there anything else one can do?

**Comment by sailod** [comment]
anyone has any suggestion what can be done to sort this out?

**Reply by Hami-d-Raza** [reply, marked as answer]
Carefully read the error message. It might provide specific details about the nature of the ban or restriction. Common error messages might include terms like "banned", "permission denied", or "access restricted".
Ensure that you have the necessary permissions to push changes to the repository. If the repository is owned by someone else, you need to have write access.

Please mark it as answered. It will help me.

**Reply by sailod** [reply]
it doesn't provide any useful information:
➜  ccxt git:(master) ✗ git push
remote: Your account is suspended. Please visit https://support.github.com for more information.
and its a fork that the push goes to my repo so there shouldn't be any limitation

**Reply by Hami-d-Raza** [reply]
You should visit the GitHub support center.

**Comment by asishkumar-gouda** [comment]
Hey @sailod  try verifying the git credentials and the forked repo should have a collaborator access too please check with the higher ups or any senior. 

If this might help !!

**Comment by Rudra-Sankha-Sinhamahapatra** [comment]
@sailod i think you trying to push to the original repo instead of your forked repo.Everytime forking isnt enough ..maybe your code is getting pushed to the original repo.So there is one way ...download github dekstop then import your forked repo(clone it or add existing) to your local machine ..then open it on your preffered ide(ex: vs code) then try to push via git bash cli 
This might help you

**Reply by sailod** [reply]
thanks for the help but Iam pretty sure Iam pushing to the forked, Ive tried now again to clone from the forked http url and also the git remote -v command show the URL with path of the forked, why should I expect to have different behavior with github desktop?

**Comment by sailod** [comment]
thanks all, I was able to push after authenticating with ssh
